### PR TITLE
Load notes by week + get project to build

### PR DESCRIPTION
**Pull request by halfcaffzach**
Added functionality to load in notes for a user on a given week
Merged in @josemdelvalle's changes
Added methods that return null to features not yet implemented so that the project could run
Added missing dependency so that project could run

### CODE REVIEW

File: JavaAPI/src/main/java/app/App.java [hunk 1]
```diff
@@ -29,7 +29,8 @@ private static void establishRoutes(Javalin app) {
         app.post("/assessments/:id/",ac.insertGrade);
         app.put("/assessments/:id/",ac.adjustWeight);
         app.post("/types",ac.createAssessmentType);
-        app.put("/assessments/:id/",ac.assignAssessmentType);
+        //TODO Give this a new name so it doesn't explode
```
Comment (halfcaffzach) [inline JavaAPI/src/main/java/app/App.java:32]: We can't have two put request with the same resource location.

File: JavaAPI/src/main/java/app/App.java [hunk 2]
```diff
@@ -29,7 +29,8 @@ private static void establishRoutes(Javalin app) {
...
+        //app.put("/assessments/:id/",ac.assignAssessmentType);
```
Comment (halfcaffzach) [inline JavaAPI/src/main/java/app/App.java:33]: comment above actually meant for this line

File: JavaAPI/src/main/java/app/App.java (hunk 1 above)
Comment (JGay90) [reply JavaAPI/src/main/java/app/App.java:32]: I hadn't caught that but your right

**Review by JGay90 — APPROVED** [review]
Looks good and very true what you said about the one I hadn't caught it

**Pull request merged**